### PR TITLE
hid placeholder on Landing Page, added image so it didn't look weird

### PR DESCRIPTION
**Pull request by Cara-Barter**
hid all placeholders on Landing Page.  Added image so that the footer wasn't crowding the NavBar

### CODE REVIEW

File: frontend/app_v2/src/components/LandingPage/LandingPagePresentation.js [hunk 1]
```diff
@@ -52,7 +51,7 @@ function LandingPagePresentation({ data }) {
       <footer>
         <Footer.Presentation />
       </footer>
```
Comment (sarahfirstvoices) [inline frontend/app_v2/src/components/LandingPage/LandingPagePresentation.js:53]: The first two footer links are broken on this. Is there an easy way to fix or hide that for this release? I don't think it's a deal breaker

Comment (Cara-Barter) [reply frontend/app_v2/src/components/LandingPage/LandingPagePresentation.js:53]: weird.  It's that same footer component that is used on the rest of the site.

Comment (sarahfirstvoices) [reply frontend/app_v2/src/components/LandingPage/LandingPagePresentation.js:53]: It looks like the urls are expecting a site value that is undefined on the landing page

**Comment by Cara-Barter** [comment]
FW-3697 Hide Landing Page placeholder

**Review by sarahfirstvoices — APPROVED** [review]

**Pull request merged**